astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,69 @@
+---
+publish_date: 2023-10-19
+title: I Just Want Authentication To Work
+description: When I am writing an Application the last thing that I care about is setting up Authentication, I just want it to work!
+authors: Alex Patterson
+image: 
+categories: Company, News
+tags: award, security mag
+excerpt_separator: "{/* more */}"
+---
+
+Authentication is the process of verifying the identity of a user or entity. It is a fundamental security measure that is used in a wide variety of applications, including web applications, mobile apps, and APIs. The thing is I just don't care about figuring it out! I just want it to work!!
+
+{/* more */}
+
+There are many different ways to implement authentication, but the most common approach is to use a username and password. Yep, just like back in 1960 when [The Everly Brothers](https://youtu.be/KtY3L5tDN_E?si=BATB3BzJJFpWsHmB) had one of the top songs out there, this method is still most popular today. Just as the Everly Brothers' time at the top of the charts has passed, so has the time for passwords. I don't want to hear about them any more!
+
+A more modern approach is to use a single sign-on (SSO) solution. SSO solutions allow users to log in to multiple applications using a single set of credentials. This can be convenient for users, as they do not have to remember multiple usernames and passwords. This method came about around the same time Eminem hit the [billboard top 100](https://youtu.be/6mPCy9LMxTI?si=5A2I-N1neb_8rGOl) for the first time. SSO often has the problem of including the first login requirement of using a username and password still to this day. Even Eminem made some changes and remains relevant!
```

Comment:
SSO often has the problem of including the first l ... This transition could be smoother.  Is this still a problem? Is it fixed?  Maybe something like this.  SSO is not perfect.  For example it has the problem of including the first log.....to this day.  However, just as Eminem has made some changes to stay relevant, so is SSO.